### ISSUE TITLE
installing nodejs on freebsd 10.1 vanilla ..

### ISSUE THREAD

**Issue by bennisixx** [atom/node-chromium-pickle#2]
It stops here, figured Id pass the info

npm ERR! FreeBSD 10.1-RELEASE-p10
npm ERR! argv "node" "/usr/local/bin/npm" "install" "chromium-pickle"
npm ERR! node v0.12.6
npm ERR! npm  v2.12.1
npm ERR! code ELIFECYCLE

npm ERR! chromium-pickle@0.1.4 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the chromium-pickle@0.1.4 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the chromium-pickle package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls chromium-pickle
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /usr/home/ben/bin/atom/npm-debug.log

 info it worked if it ends with ok
1 verbose cli [ 'node', '/usr/local/bin/npm', 'install', 'chromium-pickle' ]
2 info using npm@2.12.1
3 info using node@v0.12.6
4 verbose install initial load of /usr/home/ben/bin/atom/package.json
5 verbose readDependencies loading dependencies from /usr/home/ben/bin/atom/package.json
6 silly cache add args [ 'chromium-pickle', null ]
7 verbose cache add spec chromium-pickle
8 silly cache add parsed spec { raw: 'chromium-pickle',
8 silly cache add   scope: null,
8 silly cache add   name: 'chromium-pickle',
8 silly cache add   rawSpec: '',
8 silly cache add   spec: '*',
8 silly cache add   type: 'range' }
9 silly addNamed chromium-pickle@*
10 verbose addNamed "*" is a valid semver range for chromium-pickle
11 silly addNameRange { name: 'chromium-pickle', range: '*', hasData: false }
12 silly mapToRegistry name chromium-pickle
13 silly mapToRegistry using default registry
14 silly mapToRegistry registry https://registry.npmjs.org/
15 silly mapToRegistry uri https://registry.npmjs.org/chromium-pickle
16 verbose addNameRange registry:https://registry.npmjs.org/chromium-pickle not in flight; fetching
17 verbose request uri https://registry.npmjs.org/chromium-pickle
18 verbose request no auth needed
19 info attempt registry request try #1 at 00:34:15
20 verbose request id 6ad8d3311977a3c6
21 http request GET https://registry.npmjs.org/chromium-pickle
22 http 200 https://registry.npmjs.org/chromium-pickle
23 silly get cb [ 200,
23 silly get   { server: 'CouchDB/1.5.0 (Erlang OTP/R16B03)',
23 silly get     etag: '"6BQU1F5MG5T1K1MKNGQS8V3GZ"',
23 silly get     'content-type': 'application/json',
23 silly get     'cache-control': 'max-age=60',
23 silly get     'content-length': '9695',
23 silly get     'accept-ranges': 'bytes',
"/usr/home/ben/bin/atom/npm-debug.log" 500L, 34653C

**Comment by AlicanC** [comment]
Same here.

`node -v`: v4.0.0
`npm -v`: 3.3.3
`node-gyp -v`: v3.0.2

**Comment by AlicanC** [comment]
Any updates on this? I can't use `asar` and `gulp-asar` because of this.

**Comment by bennisixx** [comment]
I don't think anyone else sees this post. I'm going to try fresh install freebsd 10.02 and node this weekend.

**Comment by AlicanC** [comment]
Are you sure it's about your FreeBSD installation? I am actually getting this on `Microsoft Windows 8.1 Enterprise 6.3.9600.0`.

I have a relatively clean installation of `Microsoft Windows Server 2012 R2 Datacenter 6.3.9600.0` sitting in a VM. I should install required stuff on it and try there.

Maybe you have a FreeBSD issue and I just messed up my installation.

**Comment by bennisixx** [comment]
No, I'm not sure, I have a feeling it is node-gyp or pickle, but my OS is a VM so that's easy to test

**Comment by AlicanC** [comment]
Well, I tried in my VM and no luck:

https://gist.github.com/AlicanC/8ee4d1a423484113462f

**Comment by bennisixx** [comment]
Works fine on my debian computers. I think it is operating system specific.
